cleverhans/attacks.py

```diff
@@ -172,6 +172,27 @@ def generate_np(self, x_val, **kwargs):
 
         return self.sess.run(x_adv, feed_dict)
 
+    def get_labels(self, x, kwargs):
```

Comment:
maybe rename to `get_or_guess_labels` since it can use the predictions instead of the labels?